10_deploy_rook.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/bash
+
```

Comment:
We need to source common.sh here so the KUBECONFIG variable is set, ref https://github.com/openshift-metalkube/dev-scripts/pull/177/

Reply:
Also probably set -eux similar to other scripts?